app/src/main/java/com/glia/exampleapp/MainFragment.java

```diff
@@ -377,7 +378,13 @@ private void clearSession() {
     }
 
     private void showVisitorCode() {
```

Comment:
As far as I understand there is currently no default value for the Visitor Context and I can't see anything being set for it in the example app. Or did I miss it somehow?

Reply:
Default value is null as it was before